internal/services/fabric/registration.go

```diff
@@ -0,0 +1,33 @@
+package fabric
+
+import (
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+)
+
+type Registration struct{}
+
+var _ sdk.TypedServiceRegistration = Registration{}
+
```

Comment:
Regarding the comment I left on the generated file for the github issue labeler, I think you can add another method here where you can set the label that applies to resources in this service which should _hopefully_ mean that the issue labeler file gets generated correctly.

Reply:
Thank you very much for your discovery. Indeed, the labeler file could be generated correctly after appending the `AssociatedGitHubLabel` method.